fastapi_crudrouter/core/tortoise.py

```diff
@@ -69,12 +69,7 @@ async def route(pagination: PAGINATION = self.pagination) -> List[Model]:
 
     def _get_one(self, *args: Any, **kwargs: Any) -> CALLABLE:
         async def route(item_id: int) -> Model:
-            model = await self.db_model.filter(id=item_id).first()
-
-            if model:
-                return model
-            else:
-                raise NOT_FOUND
+            return await self.db_model.get(id=item_id)
```

Comment:
```suggestion
            model = await self.db_model.get(id=item_id)

            if model:
                return model
            else:
                raise NOT_FOUND
```

How about doing like so to not change the error message?

Reply:
No, this will not work. Tortoise will raise a tortoise.exceptions.DoesNotExist error when using get if no element is found. 
In fact there are two different scenarios: 
First scenario:
`register_tortoise(..., add_exception_handler=False)` => In that case we must catch the exception
`register_tortoise(..., add_exception_handler=True)` => In that case the exception is caught by tortoise and the error message is given by tortoise.